.github/workflows/release.yaml

```diff
@@ -18,7 +18,7 @@ jobs:
         name: Set up Go
         uses: actions/setup-go@v2
         with:
-          go-version: 1.15
+          go-version: "1.16"
```

Comment:
Nice catch, I guess we update the dockerfile and forgot about the release.yaml, please remove the `"` around 1.16

Reply:
Ah, the quotes are intentional to keep YAML from interpreting the version as a number. If you look at the [actions/setup-go docs](https://github.com/actions/setup-go#usage), all the versions are quoted.

Reply:
Ahh right the american decimal solution... Then we keep it this way, thanks.